### PR TITLE
feat: add svn completion spec

### PR DESCRIPTION
**Pull request by sh-sabbir**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature

**What is the current behavior? (You can also link to an open issue here)**
Currently not available

**What is the new behavior (if this is a feature change)?**
Partial support for SVN auto-completion

**Additional info:**
Added support for a few subcommands. But I'll continue improving the spec by adding more properties to the spec

### CODE REVIEW

File: src/svn.ts [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+  description: "The first content tracker",
+  subcommands: [
+    {
+      name: ["help", "h"],
+      description: "Show help for svn",
+      args: {
+        name: "subcommand",
+        description: "Help about specific subcommand",
+        isOptional: true,
+        suggestions: subcommandsList,
```
Comment (fedeci) [inline src/svn.ts:238]:
```suggestion
        template: ["help"],
```

Comment (fedeci) [inline src/svn.ts:238]:
```suggestion
        template: ["help"],
```

File: src/svn.ts [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+          priority: 100,
+        },
+        ...globalOptions,
+      ],
+    },
+  ],
+  options: [
+    {
+      name: "--version",
+      isPersistent: false,
```
Comment (fedeci) [inline src/svn.ts:278]:
```suggestion
```

File: src/svn.ts [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+      name: "info",
+      description: "Show information about a local or remote item",
+    },
+    {
+      name: ["checkout", "co"],
+      description: "Check out a working copy from a repository",
+      args: {
+        name: "repository",
+        description: "The repository you want to checkout",
+        isOptional: false,
```
Comment (fedeci) [inline src/svn.ts:255]:
```suggestion
```

**Review by fedeci — CHANGES_REQUESTED** [review]
A couple nits and it is good to be shipped!

File: src/svn.ts (hunk 1 above)
Comment (fedeci) [reply src/svn.ts:238]: It returns all the sibling subcommands but appears to be broken rn. I'm fixing it though so you can safely add it!

**Review by mschrage — APPROVED** [review]

**Pull request merged**